doc/roots.md

```diff
@@ -129,153 +149,140 @@ x, f(x)
 ```
 
 The latter shows that zeros need not be simple zeros (i.e. $f'(x) = 0$,
-if defined) to be found. For the higher-order methods, there is a
-tolerance that can be specified so that a value is returned as a zero
-if `abs(f(x)) < tol`. The default method for `fzero` uses a very
-strict tolerance for this, otherwise defaulting to an error that at
-times might be very close to the actual zero. For this problem it
-finds the exact value:
-
-```
-x = fzero(f, 2)
-x, f(x)
-```
+if defined) to be found.
 
-But not for a similar problem:
+To investigate the algorithm and its convergence, the argument
+`verbose=true` may be specified.
 
-```
-fzero(x -> x^6, 1)
-```
 
-(Though the answer is basically on track, the algorithm takes too long
-to improve itself to the very stringent range set. For problems where
-a bracket is found, this dithering won't happen.)
+For some functions, adjusting the default tolerances may be necessary
+to achieve convergence. These include `abstol` and `reltol`, which are
+used to check if `norm(f(x)) <= max(abstol, norm(x)*reltol)`;
+`xabstol`, `xreltol`, to check if `norm(x1-x0) <= max(xabstol,
+norm(x1)*xreltol)`; and `maxevals` and `maxfnevals` to limit the
```

Comment:
On counts steps in the algorithm, one function evaluations. (Some steps have 2 or more function calls.)